.github/workflows/pr_review.yml

```diff
@@ -0,0 +1,36 @@
+name: Detekt check
+on: pull_request
+
+jobs:
+    detekt:
+        runs-on: macos-latest
+        steps:
+            -   name: Checkout repository
+                uses: actions/checkout@v2
+                with:
+                    fetch-depth: 0
+            -   name: Set up JDK 11
+                uses: actions/setup-java@v2
+                with:
+                    distribution: 'temurin'
+                    java-version: 11
+
+            -   name: Setup Gradle
+                uses: gradle/gradle-build-action@v2
+
+            -   name: Set up reviewdog
+                uses: reviewdog/action-setup@v1
+
+            -   name: Run check
+                run: ./gradlew detekt -i
+
+            -   name: 'Upload report'
```

Comment:
we may consider adding 
          retention-days: ${{ env.screenshots-artifacts-retention-days }}
to not hold those for too long

Reply:
It is kept for 30 days atm based on our organization config. Do we really want to modify it?